api/namex/constants/__init__.py

```diff
@@ -103,6 +103,12 @@ class EntityTypes(AbstractEnum):
     XPRO_SOCIETY = 'XSO'
     # Used for mapping back to legacy oracle codes
     FIRM = 'FIRM'
+    # Continued In Types
+    CONTINUE_IN = 'C'             # Continued In BC Ltd.
+    CONTINUE_IN_CBEN = 'CBEN'     # Continued In Benefit Company 
```

Comment:
Should be `CONTINUE_IN_BEN`.

I had suggested some different labels to stay consistent with the UIs but I guess it doesn't matter.

PS Small typo in line 109.

Reply:
Resolving for now. Will review new code changes instead.